mp/game/dab/scripts/weapon_mossberg.txt

```diff
@@ -1,7 +1,7 @@
 WeaponData
 {
 	// Weapon characteristics:
-	"Damage"				"11"
+	"Damage"				"10"

```

Comment:
That's not gonna be enough.
At Spread 0.0658 and Damage 7 it was still a bit more OP than it used to. Tested that recently.

Reply:
since you found the issue, this comment is irrelevant now. resolving conversation